install/tools/setupMeshEx.sh

```diff
@@ -20,24 +20,27 @@
 # Helper functions for extending the mesh with external VMs.
 
 # Script can be sourced in other files or used from tools like ansible.
-# Currently the script include helpers for GKE, other providers will be added as we
-# test them.
+# Currently the script include helpers for GKE, other providers will be added as
+# they are contributed and we test them.
 
 # Environment variables used:
 #
-# ISTIO_STAGING - directory where istio artifacts are downloaded, default to current dir
-# ISTIO_VERSION - istio version.
-# ISTIO_NAMESPACE - defaults to istio-system, needs to be set for custom deployments
+# ISTIO_STAGING - directory where istio artifacts are downloaded, default to
```

Comment:
ISTIO_STAGING is no longer being used given this PR. Perhaps remove this line.

Reply:
thx ! still cleaning up as I go but good catch